dashboards/controller-manager.libsonnet

```diff
@@ -30,7 +30,7 @@ local singlestat = grafana.singlestat;
           legend_alignAsTable=true,
           legend_rightSide=true,
         )
-        .addTarget(prometheus.target('sum(rate(workqueue_adds_total{%(clusterLabel)s="$cluster", %(kubeControllerManagerSelector)s, instance=~"$instance"}[5m])) by (instance, name)' % $._config, legendFormat='{{instance}} {{name}}'));
+        .addTarget(prometheus.target('sum(rate(workqueue_adds_total{%(clusterLabel)s="$cluster", %(kubeControllerManagerSelector)s, instance=~"$instance"}[5m])) by (cluster, instance, name)' % $._config, legendFormat='{{cluster}} {{instance}} {{name}}'));
```

Comment:
It is possible that someone uses a different name for the label that identifies a cluster. That is why this label name is customized in `config.libsonnet`.

The right thing to do here is to use `%(clusterLabel)s` at all places where you added `cluster`. You can see that strategy used on the query already 🙂

Reply:
Thanks for pointing it out! I've made the required changes.